<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>进程列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>窗口信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>561</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(112, 255, 245)</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>471</width>
        <height>431</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="widget">
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <attribute name="title">
        <string>桌面宠物设置</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>201</width>
          <height>221</height>
         </rect>
        </property>
        <property name="title">
         <string>显示模式</string>
        </property>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>89</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>点击穿透</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>89</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>半透明</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>61</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>置顶</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>98</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>隐藏</string>
         </property>
        </widget>
        <widget class="QSlider" name="horizontalSlider">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>161</width>
           <height>20</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>40</y>
           <width>21</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string> %</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>40</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>100</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_5">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>100</y>
           <width>61</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>置底</string>
         </property>
        </widget>
        <widget class="QSlider" name="horizontalSlider_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>161</width>
           <height>20</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>160</y>
           <width>61</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>大小</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>160</y>
           <width>21</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string> %</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>160</y>
           <width>31</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>10</y>
          <width>221</width>
          <height>211</height>
         </rect>
        </property>
        <property name="title">
         <string>运动模式</string>
        </property>
        <widget class="QRadioButton" name="radioButton">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>71</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>静止</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>91</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>随机运动</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>91</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>贴壁</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>130</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>x,y</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>复位</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>现在动作</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>现在位置</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>150</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>action</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>340</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>删除分身</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>340</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>添加分身</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>270</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>现在人物数：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>270</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>240</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;./character/Ptilopsi&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>230</y>
          <width>71</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>选择路径</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>290</y>
          <width>231</width>
          <height>81</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>300</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>选中人物：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>300</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Ptilopsi</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>260</y>
          <width>71</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>更改当前</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>时钟设置</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_11">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>20</y>
          <width>93</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_7">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>61</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>隐藏</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>最小化到托盘</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TestLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>壁纸设置</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumain">
    <property name="title">
     <string>main</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuout">
    <property name="title">
     <string>out</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuexit">
    <property name="title">
     <string>exit</string>
    </property>
   </widget>
   <addaction name="menumain"/>
   <addaction name="menuout"/>
   <addaction name="menuexit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
